repos/fdbt-site/tests/pages/api/searchOperators.test.ts

```diff
@@ -394,7 +398,76 @@ describe('searchOperators', () => {
             selectedOperators: mockSelectedOperators,
         };
 
-        searchOperators(req, res);
+        await searchOperators(req, res);
+
+        expect(updateSessionAttributeSpy).toHaveBeenCalledWith(
+            req,
+            MULTIPLE_OPERATOR_ATTRIBUTE,
+            expectedSessionAttributeCall,
+        );
+        expect(res.writeHead).toBeCalledWith(302, {
+            Location: '/saveOperatorGroup',
+        });
+    });
+    it("should redirect with errors when the user clicks the 'Confirm operators and continue' button and Bods has no services", async () => {
+        jest.spyOn(auroradb, 'operatorHasBodsServices').mockResolvedValue(false);
+        const mockSelectedOperators: Operator[] = [
+            { nocCode: 'BLAC', name: 'Blackpool Transport' },
+            { nocCode: 'LNUD', name: 'The Blackburn Bus Company' },
+        ];
+        const { req, res } = getMockRequestAndResponse({
+            body: {
+                continueButtonClick: 'Continue',
+                searchText: '',
+            },
+            session: {
+                [MULTIPLE_OPERATOR_ATTRIBUTE]: {
+                    selectedOperators: mockSelectedOperators,
+                },
+            },
+        });
+        const expectedSessionAttributeCall: MultipleOperatorsAttributeWithErrors = {
+            selectedOperators: mockSelectedOperators,
+            errors: [
+                { errorMessage: 'Some of the selected operators have no services', id: 'remove-operator-checkbox-0' },
+                { errorMessage: 'Blackpool Transport', id: 'remove-operator-checkbox-0' },
+                { errorMessage: 'The Blackburn Bus Company', id: 'remove-operator-checkbox-0' },
+            ],
+        };
+
+        await searchOperators(req, res);
+
+        expect(updateSessionAttributeSpy).toHaveBeenCalledWith(
+            req,
+            MULTIPLE_OPERATOR_ATTRIBUTE,
+            expectedSessionAttributeCall,
+        );
+        expect(res.writeHead).toBeCalledWith(302, {
+            Location: '/searchOperators',
+        });
+    });
+    it("should redirect to /saveOperatorGroup when clicking  'Confirm operators and continue' button and Bods has services", async () => {
```

Comment:
Similarly here,

it("should redirect to /saveOperatorGroup when clicking  'Confirm operators and continue' button and all of the operators have bods services",